app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Users::PasswordsController < Devise::PasswordsController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.